iceberg/iceberg-handler/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergStorageHandlerNoScan.java

```diff
@@ -239,6 +240,36 @@ public void testCreateDropTableNonDefaultCatalog() throws TException, Interrupte
     );
   }
 
+  @Test
+  public void testCreateTableStoredByIceberg() {
+    TableIdentifier identifier = TableIdentifier.of("default", "customers");
+    String query = String.format("CREATE EXTERNAL TABLE customers (customer_id BIGINT, first_name STRING, last_name " +
+        "STRING) STORED BY %s %s TBLPROPERTIES ('%s'='%s')",
+        "ICEBERG",
```

Comment:
I would change most of the test to `STORED BY ICEBERG` and keep only a few with the original class name.
Also I would like to try out `stored by iCeBerG`

Reply:
Done.